vms/event/views.py

```diff
@@ -217,13 +217,43 @@ def post(self, request, *args, **kwargs):
                     return HttpResponseRedirect(reverse('event:list'))
             else:
                 data = request.POST.copy()
+                try:
+                    country_name = data.get('country')
+                    country = Country.objects.get(name=country_name)
+                except ObjectDoesNotExist:
+                    country = None
+                try:
+                    state_name = data.get('state')
+                    state = Region.objects.get(name=state_name)
+                except ObjectDoesNotExist:
+                    state = None
+                try:
+                    city_name = data.get('city')
+                    city = City.objects.get(name=city_name)
+                except ObjectDoesNotExist:
+                    city = None
+                for field in data.keys():
+                    try:
+                        data[field] = data.get(field)
```

Comment:
just do `data.get(field, '')`

Reply:
@SanketDG please review it again. Thanks.